proxies/go/handler_command.go

```diff
@@ -35,5 +58,62 @@ func commandHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	json.NewEncoder(w).Encode(body)
+	params := parseParams(body)
+	entity := getEntityFromLocation(location, id)
+	var method reflect.Value
+
+	if locationIsClient {
+		apiClient := entity.(devcycle.DVCClient).DevCycleApi // client api methods are called on inner DevCycleApi object
+		auth := datastore.clientAuthContexts[id]
+		params = append([]reflect.Value{reflect.ValueOf(auth)}, params...)
+		method = reflect.ValueOf(apiClient).MethodByName(strings.Title(body.Command))
+	} else {
+		method = reflect.ValueOf(entity).MethodByName(strings.Title(body.Command))
+	}
+
+	result := method.Call(params)
+
+	entityType := method.Type().Out(0).Name()
+	if entityType != "Variable" {
+		entityType = "Object"
+	}
+
+	response := LocationResponse{
+		entityType,
+		result[0].Interface(),
+		make([]string, 0),
+	}
+
+	locationId := strconv.Itoa(len(datastore.commandResults[body.Command]))
+	locationHeader := "command/" + body.Command + "/" + locationId
+	datastore.commandResults[body.Command][locationId] = result[0]
+
+	w.Header().Set("Location", locationHeader)
+	json.NewEncoder(w).Encode(response)
+
+	// TODO handle exception and asyncError
+	// TODO test with local bucketing
+}
+
+func getEntityFromLocation(location string, id string) any {
+	if location != "" {
```

Comment:
seems like this is somewhat duplicating the logic above that's determining whether its a user request

Reply:
good point, i added locationIsClient as a param